backend/controller/pubsub/integration_test.go

```diff
@@ -34,13 +35,14 @@ func TestPubSub(t *testing.T) {
 				WHERE
 					state = 'success'
 					AND origin = '%s'
-		`, dal.AsyncOriginPubSub{Subscription: schema.RefKey{Module: "subscriber", Name: "testSubscription"}}.String()),
+		`, dal.AsyncOriginPubSub{Subscription: schema.RefKey{Module: "subscriber", Name: "testTopicSubscription"}}.String()),
```

Comment:
I renamed this one so the Java test could use the auto generated subscription annotation, I left the other the same to allow the use of an explicit name.